jkube-kit/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmServiceUploadIT.java

```diff
@@ -481,6 +485,54 @@ void withSuccessfulUpload_shouldUploadBlobsAndUpdateManifest() throws Exception
             "  } ]\n" +
             "}");
     }
+    @Test
+    @DisplayName("On Windows, Sends chart metadata file, tarball and manifest file in separate requests")
+    @EnabledOnOs(OS.WINDOWS)
+    void withSuccessfulUploadOnWindows_shouldUploadBlobsAndUpdateManifest() throws Exception {
+      helmConfig.setChartExtension("tar.gz");
+      Files.write(
+          Paths.get(helmConfig.getOutputDir()).resolve("kubernetes")
+              .resolve("Helm-Chart-1337-SNAPSHOT.tar.gz"),
+          "I'm a tar.gz, not a .tgz".getBytes(StandardCharsets.UTF_8));
+      Files.write(Paths.get(helmConfig.getOutputDir()).resolve("kubernetes").resolve("Chart.yaml"),
+          "---\r\napiVersion: v1\r\nname: test-chart\r\nversion: 0.0.1".getBytes(StandardCharsets.UTF_8));
```

Comment:
rather than using platform-specific line separators, could you please use `String.format` with `%n`?

Reply:
actually, I purposefully used platform-specific line separators because this test was exclusively created for windows Os. Should I still use String.format() with %n?

Reply:
I would try to reduce platform specific code even if this test is enabled only on windows. I think the reason for creating separate test was because of difference in the digest value and size, right?

Reply:
yeah. That was the main reason for the test failure. No issues, I'll make the recommended changes and push the commit in a few minutes.

Reply:
Should I make the same changes on the other test?